backend/clientConnection.js

```diff
@@ -0,0 +1,15 @@
+// Will use .env in the future. For now using this.
+
+const { Client } =  require("pg")
```

Comment:
This looks like you're using [node-postgres](https://www.npmjs.com/package/pg), but you also have [Prisma](https://prisma.io) migrations and a schema. I'm a little confused why you're using both.

Reply:
Looking further, it seems like this isn't really being used and you are using Prisma. Can this file be deleted?

Reply:
You are correct, the clientConnection.js file can be removed. I didn't pay attention to it before.